src/components/FamilyManifestModal/queries.js

```diff
@@ -1,6 +1,38 @@
 import { get, difference } from 'lodash';
 import graphql from 'services/arranger';
 
+export const participantsFilesCountAndSize = async ({ api, sqon }) => {
+  const response = await graphql(api)({
+    query: `
+        query familyMemberAndParticipantData($sqon: JSON, $include_missing: Boolean) {
+          file {
+
+            hits(filters: $sqon) {
+              total
+            }
+
+            aggregations(filters: $sqon, include_missing: $include_missing) {
+              size {
+                stats {
+                  sum
+                }
+              }
+            }
+
+          }
+        }
+      `,
+    variables: { sqon, include_missing: true },
+  });
+
+  const { data } = response;
+
+  const participantFilesCount = get(data, 'file.hits.total') || 0;
+  const participantFilesSize = get(data, 'file.aggregations.size.stats.sum') || 0;
```

Comment:
I wouldn't bother unpacking `{ data }` above if using `get`, also get defaults = third arg
```
return { participantFilesCount: get(data, 'data.file.hits.total', 0), ... }
```
also with that change I wouldn't include the intermittent vars, just construct the json in the return statement